src/custom/state/orders/updaters/UnfillableOrdersUpdater.ts

```diff
@@ -38,11 +40,17 @@ async function _getOrderPrice(chainId: ChainId, order: Order, strategy: GpPriceS
     quoteToken = order.sellToken
   }
 
+  const isNativeSellSwap = order.sellToken === NATIVE_CURRENCY_BUY_ADDRESS
```

Comment:
@alfetopito pinging u here 

we need to check unfillables with the wrapped address (when ethflow) so this code makes sense?

Reply:
I don't fully get the question. Can you give me a bit more context?

Reply:
Looking at the code gave me more context :)

Now that you mention I wonder. Wouldn't we need to have this for when buying ETH too?

Other than that, yes, it makes sense